admin-dev/themes/new-theme/js/pages/product/components/dropzone/Dropzone.vue

```diff
@@ -494,7 +494,7 @@
           const imageElement = document.querySelector(
             DropzoneMap.savedImage(newImage.image_id),
           );
-          var imageUpdateTime = new Date();
+          const imageUpdateTime = new Date();
           imageElement.src = newImage.image_url + '?' + imageUpdateTime.getTime();
```

Comment:
```suggestion
          imageElement.src = `${newImage.image_url}?${imageUpdateTime.getTime()}`;
```

Damn, I don't now why eslint didn't cry before